File: src/main/java/takumi3s/rcdamagecore/config/RDCConfig.java
package takumi3s.rcdamagecore.config;

import org.bukkit.configuration.ConfigurationSection;

public class RDCConfig {
    public static int playerMaxLevel = 0;

    public static void loadConfig(ConfigurationSection section) {
        playerMaxLevel = section.getInt("playerMaxLevel", 10);
    }
}

File: src/main/kotlin/net/azisaba/rcdamagecore/config/RDCConfig.kt
package net.azisaba.rcdamagecore.config

import org.bukkit.configuration.ConfigurationSection

object RDCConfig {
    private var playerMaxLevel: Int = 10
    private var levelGapAdjustment: Int = 80

    fun getPlayerMaxLevel(): Int = playerMaxLevel

    fun getLevelGapAdjustment(): Int = levelGapAdjustment

    fun loadConfig(section: ConfigurationSection) {
        playerMaxLevel = section.getInt("playerMaxLevel", 10)
        levelGapAdjustment = section.getInt("levelGapAdjustment", 80)
    }
}
